tests/__init__.py

```
@@ -63,6 +63,71 @@
+			MAGENTA = """\033[35m"""
+			CYAN = """\033[36m"""
+			GREY = """\033[37m"""
+			AMBER = """\033[93m"""
+			REDBG = """\033[41m"""  # Red background
+			ENDC = """\033[0m"""
+
+		logging_color = {
+			'debug': ANSIColors.BLUE, 'info': ANSIColors.GREY,
+			'warning': ANSIColors.AMBER,
+			'error': ANSIColors.RED,
+			'critical': str(str(ANSIColors.BLACK) + str(ANSIColors.REDBG))
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)